frontend/demo/component/board/example-indicator.ts

```diff
@@ -42,9 +43,9 @@ export class ExampleIndicator extends LitElement {
           font-size: var(--lumo-font-size-m);
         }
 
-        .icon iron-icon {
-          --iron-icon-width: var(--lumo-font-size-xs);
-          --iron-icon-height: var(--lumo-font-size-xs);
+        .icon vaadin-icon {
+          --vaadin-icon-width: var(--lumo-font-size-xs);
+          --vaadin-icon-height: var(--lumo-font-size-xs);
```

Comment:
`vaadin-icon` doesn't have this CSS variable, use plain width/height instead

```suggestion
          width: var(--lumo-font-size-xs);
          height: var(--lumo-font-size-xs);
```